### DISCUSSION TITLE
Do I need a GPU to run Habitat-sim?

### DISCUSSION THREAD

**Discussion by hongruihuang** [facebookresearch/habitat-lab#1454] [Q&A]
If I don't have gpu,is it possible for me to use this simulator sucessfully?

**Comment by aclegg3** [comment]
Yeah, you can use Habitat locally without a GPU.

**Reply by Neynix07** [reply]
May I know to use it without GPU? I tried on WSL2 and successfully installed habitat-sim version 0.3.3. When I tested with the data, an error occurs:
Traceback (most recent call last):
  File "/home/neynix/habitat-sim/examples/viewer.py", line 1204, in <module>
    HabitatSimInteractiveViewer(sim_settings).exec()
  File "/home/neynix/habitat-sim/examples/viewer.py", line 64, in __init__
    self.Configuration(
TypeError: __init__(): incompatible constructor arguments. The following argument types are supported:
    1. _magnum.platform.glfw.Application.Configuration()

Invoked with: kwargs: title='Habitat Sim Interactive Viewer', size=(800, 600)
